libr/asm/d/Makefile

```diff
@@ -1,4 +1,4 @@
-FILES=6502 8051 m68k x86 arc arm avr bpf LH5801 ppc z80 mips sparc sh malbolge xtensa
```

Comment:
dont move the asm/d, this is part of asm/ not anal/

Reply:
Sorry I don't understand, I thought the  plugin should be removed from `asm/d`?

Reply:
Oh, I get it now 🤦🏻